src/Gen.jl

```diff
@@ -23,9 +23,7 @@ Permit use of generative functions written in the static modeling language up
 to this point. Functions are loaded into the calling module.
 """
 macro load_generated_functions()
-    for function_defn in generated_functions
-        Core.eval(__module__, function_defn)
-    end
+    :(load_generated_functions($(__module__)))
```

Comment:
Is there a reason this change was made? This implementation depends on `load_generated_functions` still existing as a function, but we were planning to deprecate it in #234. It also _might_ lead to errors if we do `import Gen`, because `load_generated_functions` isn't referenced.

Reply:
I was getting some error with `@load_generated_functions` that this fixed.  I think it was somehow caused by the fact that the old version of the macro did the loading at macro runtime rather than function runtime, but I don't really understand what the issue was.  We wouldn't need to keep `load_generated_functions` to keep this moved to runtime; we just need to make sure we return the original loop as a quoted expression.  Might be worth understanding better what the issue I was getting is.